### PR TITLE
allow filter whatsapp data to remove some common messages

### PR DESCRIPTION
**Pull request by snopoke**
Also test RTL unicode text.

### CODE REVIEW

**Review by SmittieC — APPROVED** [review]

File: apps/analysis/tests/test_whatsapp_parser.py [hunk 1]
```diff
@@ -21,24 +22,58 @@ def valid_whatsapp_log():
...
+        "2023-07-13 15:54",
+        "123456",
+        "اولا.لسلام عليكم  : كل من كان هذه المجموعة.  بعد السلام: اشكركم\n\n  جميعا خاصة المعلمون المدرسون فى المجموعة. اتمنى  لكم النجاح.\n‏",
+    )
+    _check_message(df, "2023-07-13 16:23", "123123", "وعليكم السلام مرحبا يااخي💙🌸")
+    _check_message(df, "2023-07-13 20:28", "Coach", "Halkan baad ka daawan kartaan casharka oo muuqaal ah.")
 
 
 def _check_message(df, date, sender, message):
+    print(df.loc[pd.Timestamp(date)]["sender"])
```
Comment (snopoke) [inline apps/analysis/tests/test_whatsapp_parser.py:76]:
```suggestion
```
